wasp_pickpocketer.simba

```diff
@@ -535,6 +535,11 @@ begin
   FreshNeck := HasNeck := Result;
 end;
 
+function TThiever.WithdrawNecks(): Boolean;
+begin
+  if Result := Bank.WithdrawItem([Necklace, NeckAmount - Inventory.CountItem(Necklace), False], False) then Wait(300, 400);
+end;
```

Comment:
Updated bank withdraw to have a wait, disable using "Use X", and calculated how many to withdraw on each bank run.

There is a bug where sometimes it withdraws necklaces twice quickly. Wait time should prevent that.

Reply:
in testing it still sometimes double withdraws, but at least it withdraws a lot fewer 😆